### ISSUE TITLE
Custom Domains vs Hostnames

### ISSUE THREAD

**Issue by callumbwhyte** [umbraco/Umbraco.Cloud.Issues#825]
### Issue description

The portal seems to interchangeably use the terminology "Custom Domains" and "Hostnames" – which is often a little confusing...

The projects overview lists a count of "Custom Domains", whilst the portal section for managing them is called and refers to them as "Hostnames" everywhere (content + sidebar)

<img width="467" alt="Screenshot 2024-07-29 at 10 52 07" src="https://github.com/user-attachments/assets/70d16feb-ce02-4847-a782-814f98d2ffa7">

<img width="1728" alt="Screenshot 2024-07-29 at 10 53 11" src="https://github.com/user-attachments/assets/3b1fc91d-dae1-4ea1-9d73-708f68b9998a">

**Comment by c9mb** [comment]
Yes, it's confusing that UC interchanges those terms, particularly given that there is a limit placed on Custom Domains, each of which may have multiple Host Names, and it's the Host Names that you create entries for - so the distinction needs to be clarified.
